v2/pkg/operator/local_stored_collector.go

```diff
@@ -254,3 +255,27 @@ func (o *LocalStorageCollector) prepareM2DCopyBatch(log clog.PluggableLoggerInte
 	}
 	return result, nil
 }
+
+func isDigestName(img string) bool {
+	return strings.Contains(img, "@")
+}
+
+func imageShortName(img string) string {
```

Comment:
```suggestion
func imageName(reference string) string {
```
Not sure if this suggested name is better tbh...
According to https://github.com/containers/image/blob/main/docker/reference/reference.go what we are extracting is the name from the reference.

Reply:
Good suggestion, done!